src/core/Camera.ts

```diff
@@ -117,53 +149,64 @@ class Camera {
     this._view3D = view3D;
     this._threeCamera = new THREE.PerspectiveCamera();
     this._maxTanHalfHFov = 0;
+    this._baseFov = 45;
+    this._baseDistance = 0;
 
     const initialZoom = isNumber(view3D.initialZoom) ? view3D.initialZoom : 0;
 
     this._defaultPose = new Pose(view3D.yaw, view3D.pitch, initialZoom);
     this._currentPose = this._defaultPose.clone();
+    this._newPose = this._currentPose.clone();
   }
 
   /**
    * Reset camera to default pose
-   * @param duration Duration of the reset animation
-   * @param easing Easing function for the reset animation
+   * @param {number} [duration=0] Duration of the reset animation
+   * @param {function} [easing] Easing function for the reset animation
+   * @param {Pose} [pose] Pose to reset, camera will reset to `defaultPose` if pose is not given.
    * @returns Promise that resolves when the animation finishes
    */
-  public async reset(duration: number = 0, easing: (x: number) => number = DEFAULT.EASING): Promise<void> {
+  public reset(duration: number = 0, easing: (x: number) => number = DEFAULT.EASING, pose?: Pose): Promise<void> {
```

Comment:
Looks like `pose = this._defaultPose` parameter?

Reply:
That would be better 👍